gulpfile.js

```diff
@@ -1,25 +1,142 @@
 var gulp = require('gulp'),
   bump = require('gulp-bump'),
-  filter = require('gulp-filter'),
   git = require('gulp-git'),
   sourcemaps = require('gulp-sourcemaps'),
   tag_version = require('gulp-tag-version'),
   tslint = require('gulp-tslint'),
-  ts = require('gulp-typescript');
+  ts = require('gulp-typescript'),
+  PluginError = require('plugin-error'),
+  minimist = require('minimist'),
+  path = require('path');
 
-// compile
-gulp.task('compile', function() {
+const exec = require('child_process').exec;
+const spawn = require('child_process').spawn;
+
+const releaseOptions = {
+  semver: '',
+  githubToken: '',
+};
+
+// prettier
+function runPrettier(command, done) {
+  exec(command, function(err, stdout) {
+    if (err) {
+      return done(new PluginError('runPrettier', { message: err }));
+    }
+
+    if (!stdout) {
+      return done();
+    }
+
+    var files = stdout
+      .split(/\r?\n/)
+      .filter(f => {
+        return f.endsWith('.ts') || f.endsWith('.js');
+      })
+      .join(' ');
+
+    if (!files) {
+      return done();
+    }
+
+    const prettierPath = path.normalize('./node_modules/.bin/prettier');
+    exec(
+      `${prettierPath} --write --print-width 100 --single-quote --trailing-comma es5 ${files}`,
+      function(err) {
+        if (err) {
+          return done(new PluginError('runPrettier', { message: err }));
+        }
+        return done();
+      }
+    );
+  });
+}
+
+function createChangelog(done) {
+  const imageName = 'jpoon/github-changelog-generator';
+  const version = require('./package.json').version;
+
+  var options = minimist(process.argv.slice(2), releaseOptions);
+
+  if (!options.githubToken) {
```

Comment:
any chance you can make this take an environment variable too if the argument is not defined? It is not a big deal but I had been using CHANGELOG_GITHUB_TOKEN

Reply:
Done